tests/export-graph.sh

```diff
@@ -4,7 +4,7 @@ clearStore
 clearProfiles
 
 checkRef() {
-    nix-store -q --references $TEST_ROOT/result | grep -q "$1" || fail "missing reference $1"
+    nix-store -q --references $TEST_ROOT/result | grep -q "$1$" || fail "missing reference $1"
```

Comment:
Nit: Would it be a good idea to explicitly escape that trailing `$`? Otherwise, it kinda looks like the `%1%` pattern we have around.

Reply:
How so, like
```suggestion
    nix-store -q --references $TEST_ROOT/result | grep -q "$1"'$' || fail "missing reference $1"
```
?

Reply:
I more meant just like `"$1\$"`, but that works too :P